tree/ntuple/v7/src/RNTupleSerialize.cxx

```diff
@@ -65,27 +65,23 @@ std::uint32_t SerializeFieldV1(const ROOT::Experimental::RFieldDescriptor &field
    return size;
 }
 
-std::uint32_t SerializeFieldTree(
-   const ROOT::Experimental::RNTupleDescriptor &desc,
-   ROOT::Experimental::Internal::RNTupleSerializer::RContext &context,
-   void *buffer)
+std::uint32_t SerializeFieldList(const ROOT::Experimental::RNTupleDescriptor &desc,
+                                 std::span<const ROOT::Experimental::DescriptorId_t> fieldList,
+                                 std::size_t firstOnDiskId,
```

Comment:
Add documentation to this parameter, whose meaning at least to me was not obvious when reading the code (it became clear when seeing how the function is used).

Can we not keep this information in the context, btw? Like: serialized up to this and that id?

Reply:
> Can we not keep this information in the context, btw? Like: serialized up to this and that id?

We could, but given that these functions may be called more than once, e.g. to compute the buffer size prior to serialization, I would prefer to leave as-is.  Also, this function is considered internal.

I have documented the function though.